transcendence-app/src/user/dto/user.dto.ts

```diff
@@ -30,3 +31,15 @@ export class UserDto {
   @ValidateIf((object, value) => value !== null)
   avatarId!: string | null;
 }
+
+export class UserDto {
+  username!: string;
+  email!: string;
+  fullName!: string;
+  avatarId!: string | null;
+  avatarX!: number;
+  avatarY!: number;
+  id!: string;
+  createdAt!: Date;
+  isBlocked?: boolean | null;
```

Comment:
I don't think is a good idea. Let me explain:

The `User` domain represents a user in our problem which all the characteristics that makes a user (name, email, avatar etc.)

On the other hand, `isBlocked` is a status between users and do not represent nothing for the user itself. It's a relation between users.

We include it in this dto because it's **aggregated data** to communicate between layers (backend to frontend in this case) representing the `block` status between the user making the request and the fetched user itself and w